cluster/manifests/kube-node-ready-controller/deployment.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kube-node-ready-controller
+  namespace: kube-system
+  labels:
+    application: kube-node-ready-controller
+    version: latest
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      application: kube-node-ready-controller
+  template:
+    metadata:
+      labels:
+        application: kube-node-ready-controller
+        version: latest
+    spec:
+      tolerations:
+      - key: node.alpha.kubernetes.io/notReady-workload
+        operator: Exists
+        effect: NoSchedule
+      containers:
+      - name: kube-node-ready-controller
+        image: mikkeloscar/kube-node-ready-controller:latest
+        args:
+        - --not-ready-taint-name=node.alpha.kubernetes.io/notReady-workload
+        # format <namespace>:<labelKey>=<labelValue>,+
+        - "--pod-selector=kube-system:application=skipper-ingress"
```

Comment:
This needs to be excluded since we _don't want_ Skipper to run nodes that haven't been fully initialised.

Reply:
oops

Reply:
Fixed.